### PR TITLE
PHP decbin() math function

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create a reference for the decbin() function in PHP. Motivation emanates from a successful 2nd contribution to docs and the blossoming desire to help other learners in the field.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:25]:
```suggestion
```

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:34]:
```suggestion
```
Remove this as you've already stated below what the output will be

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
2 changes

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:8]: remove these 2. We list a max of 2 under subjects

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:19]: I would recommend moving this sentence down to the syntax section

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:39]: I would recommend removing the codebyte since it isn't demonstrating any new functionality beyond the previous example. We aim not to duplicate examples. Each example should have a purpose.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arisdelacruz I've completed a maintainer review. Please update the entry, and tag me when you've finished making the changes. Thanks

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/decbin/decbin.md:8]: noted on this

will edit accordingly

thanks

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/decbin/decbin.md:19]: this is duly noted

thanks

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/decbin/decbin.md:39]: noted on this

thanks

**Comment by arisdelacruz** [comment]
@caupolicandiaz just did another push

for your consideration, bud

thanks

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz There were still some changes that weren't quite right. I went ahead and pushed the edits. I'm sending this along for a second review.

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:3]:
```suggestion
Description: 'Converts a decimal number to a string containing a binary number.'
```

File: content/php/concepts/math-functions/terms/decbin/decbin.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/decbin/decbin.md:15]:
```suggestion
The **`decbin()`** function converts a decimal number to a string containing a binary number.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz I did a review and left a couple of minor comments.

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz looks good to merge.

**Comment by SSwiniarski** [comment]
@arisdelacruz Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/math-functions/decbin

image

**Pull request merged**
